### PR TITLE
Subscription check

### PR DESCRIPTION
**Pull request by yaksheshgupta**

### CODE REVIEW

File: modules/PrimaryPool/substreams.yaml [hunk 1]
```diff
@@ -27,13 +34,27 @@ modules:
...
+    inputs:
+      - source: sf.ethereum.type.v2.Block
+    output:
+      type: proto:verified.primary.v1.PrimaryIssues
 
   - name: kv_out
     kind: map
     inputs:
-      - map: map_subscriptions
+      - store: store_block_meta_end
```
Comment (kallolborah) [inline modules/PrimaryPool/substreams.yaml:54]: @yaksheshgupta should not this input be the subscriptions map ?

File: modules/PrimaryPool/substreams.yaml [hunk 2]
```diff
@@ -27,13 +34,27 @@ modules:
     kind: map
     initialBlock: 8586768
     inputs:
+      - source: sf.ethereum.type.v2.Block
+      # Added UP
       - map: map_pools
```
Comment (kallolborah) [inline modules/PrimaryPool/substreams.yaml:39]: @yaksheshgupta are we using map_subscriptions or map_subscriptionscheck ? One seems to be taking map pools as input - is this what you were mentioning yesterday on telegram ?

File: modules/PrimaryPool/substreams.yaml [hunk 3]
```diff
@@ -27,13 +34,27 @@ modules:
...
     output:
-      type: proto:verified.primary.v1.Subscriptions
+      type: proto:verified.primary.v1.PrimaryIssues
+
+  - name: map_subscriptionsCheck
```
Comment (kallolborah) [inline modules/PrimaryPool/substreams.yaml:43]: @yaksheshgupta are we using map_subscriptions or map_subscriptionscheck ? One seems to be taking map pools as input - is this what you were mentioning yesterday on telegram ?

**Review by kallolborah — APPROVED** [review]
@yaksheshgupta left a couple of comments after checking the code. I do not understand why we are using eth meta blocks - is it because you want the deltas of the subscription data being emitted ?

File: modules/PrimaryPool/substreams.yaml (hunk 3 above)
Comment (yaksheshgupta) [reply modules/PrimaryPool/substreams.yaml:43]: Yes "map_subscriptionscheck" is just a function that return "subscriptions" event. Similar to "map_pools".
Its just for testing purposes

File: modules/PrimaryPool/substreams.yaml (hunk 1 above)
Comment (yaksheshgupta) [reply modules/PrimaryPool/substreams.yaml:54]: Yes, it should be.
I just commented out the code for testing purpose only

**Pull request merged**